fastlane/spec/actions_specs/upload_symbols_to_crashlytics_spec.rb

```diff
@@ -0,0 +1,18 @@
+describe Fastlane do
+  describe Fastlane::FastFile do
+    describe "upload_symbols_to_crashlytics" do
+      it "generates a valid command" do
+        result = Fastlane::FastFile.new.parse("lane :test do
+          upload_symbols_to_crashlytics(
+            dsym_path: './fastlane/spec/fixtures/dSYM/Themoji.dSYM.zip',
+            api_token: 'something123',
+            binary_path: './fastlane/spec/fixtures/screenshots/screenshot1.png')
```

Comment:
It's just for the tests, since we don't want to bundle the actual binary for the test cases.